schedule/yast/encryption/crypt_no_lvm_pvm.yaml

```diff
@@ -7,12 +8,14 @@ description: >
   module is scheduled and OPT_KERNEL_PARAMS variable is set.
   In comparison to SLE 12 we register the installation and have system roles
   wizard screen.
-name: cryptlvm
 vars:
-  OPT_KERNEL_PARAMS: console=%SERIALDEV% VNCSize=1024x768
   DESKTOP: textmode
   ENCRYPT: 1
+  INSTALLONLY: 1
   LVM: 0
+  MAX_JOB_TIME: 14400
+  OPT_KERNEL_PARAMS: console=%SERIALDEV% VNCSize=1024x768
+  YAML_TEST_DATA: test_data/yast/encryption/encrypt_no_lvm.yaml
```

Comment:
I wonder if we should use this variable if defined in the schedule. It's definitely not wrong, but maybe we should discuss if we use only `test_data` node in case it's schedule file.

Reply:
+1 we created to make possible different test data for a test suite.

> LGTM+approved
> Only thing if you could fix is the commit title. Something like "migrates settings to yast jobgroup" or something like that.

it is related with the migration but just a consequence of the cleaning, I updated a bit more.